Description: Drop the URL to the SourceForge logo
 This avoids the Lintian error privacy-breach-logo.
Author: Rafael Laboissiere <rafael@debian.org>
Bug: https://savannah.gnu.org/bugs/index.php?49709
Forwarded: no
Last-Update: 2016-11-27

--- octave-4.2.0.orig/scripts/testfun/private/html_plot_demos_template.html
+++ octave-4.2.0/scripts/testfun/private/html_plot_demos_template.html
@@ -50,10 +50,6 @@
 <div id="content">
 <!-- ##ADD TABLE HERE## -->
 
-<div id="sf_logo">
-  <a href="http://sourceforge.net"><img src="http://sourceforge.net/sflogo.php?group_id=2888&amp;type=1"
-     width="88" height="31" style="border: 0;" alt="SourceForge.net Logo"/></a>
-</div>
 </div>
 </body>
 </html>
